core/templates/dev/head/domain/collection/CollectionRightsBackendApiService.js

```diff
@@ -20,24 +20,19 @@ oppia.factory('CollectionRightsBackendApiService', [
     '$http', '$q', 'COLLECTION_RIGHTS_URL_TEMPLATE', 'UrlInterpolationService',
     function($http, $q, COLLECTION_RIGHTS_URL_TEMPLATE,
       UrlInterpolationService) {
-      var _setCollectionStatus = function(
-          collectionId, collectionVersion, isPublic, successCallback,
-          errorCallback) {
+      var SetCollectionRights = function(
+        collectionId, putParams, successCallback, errorCallback) {
```

Comment:
I am thinking of splitting `CollectionRightsHandler.put` to `CollectionsPublicizeOptionsHandler.put` and `CollectionsMembersHandler.put`, what do you think?